proj/src/main/webapp/index.html

```diff
@@ -11,13 +11,26 @@
         <!-- TODO: Add SDKs for Firebase products that you want to use
             https://firebase.google.com/docs/web/setup#available-libraries -->
         <script src="https://www.gstatic.com/firebasejs/7.15.4/firebase-analytics.js"></script>
+
+        <script src="app.js"></script>
```

Comment:
Are you including a file that's gitignored?  Why?

Reply:
This is the file that has all of the private information required to run. We each have a version of the file, but aren't enthusiastic about having it publicly available

Reply:
Ah.  But then when it's deployed, the user can just fetch the script from the server and see it that way, no?  Doesn't sound like a very good security model.  I think in general it's impossible to keep information in client-side JavaScript private, which is why it's important to have server-side checks that prevent users from abusing that information.  Do we have a plan to move forward so that this information can eventually be committed without a security concern?  If not, I think we need to make one.